data/json/monstergroups/zombies.json

```diff
@@ -538,6 +538,41 @@
       { "monster": "mon_dog_zombie_cop", "weight": 50, "cost_multiplier": 2, "pack_size": [ 1, 2 ] }
     ]
   },
+  {
+    "name": "GROUP_PANICROOM",
+    "type": "monstergroup",
+    "//": "Flavor spawn group for panic rooms for better story-telling.  Improved chance of dogs, fancy dogs, and (still) breathing people and children.",
+    "monsters": [
+      { "monster": "mon_zombie_child", "weight": 180 },
+      { "monster": "mon_zombie", "weight": 120, "cost_multiplier": 2 },
+      { "group": "GROUP_CIVILIANS_STANDARD", "weight": 200 },
+      { "group": "GROUP_FERAL", "weight": 150, "cost_multiplier": 2 },
+      { "group": "GROUP_PET_DOGS", "weight": 100, "cost_multiplier": 2 },
+      { "monster": "mon_dog_rottweiler", "weight": 100, "cost_multiplier": 2 },
+      { "monster": "mon_dog_gpyrenees", "weight": 100, "cost_multiplier": 2 },
+      { "monster": "mon_zombie_dog", "weight": 20 }
+    ]
+  },
+  {
+    "name": "GROUP_FANCY_PANICROOM",
+    "type": "monstergroup",
+    "//": "Same as above, but with the fancy ferals thrown into the mix for the scenario.",
+    "monsters": [
+      { "monster": "mon_zombie_child", "weight": 250 },
```

Comment:
I'd do:

```json
{ "group": "GROUP_PANICROOM" }
```

Then remove all duplicated monsters.

Reply:
You are right, I will!